File: Desafios/Bee1046.java
import java.util.Scanner;
public class Bee1046 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int inicio = scanner.nextInt();
        int fim = scanner.nextInt();
        scanner.close();
        if (inicio < fim) {
            System.out.println("O JOGO DUROU " + (fim - inicio) + " HORA(S)");
        }
        else if (inicio == fim) {
            System.out.println("O JOGO DUROU 24 HORA(S)");
        }
        else {
            System.out.println("O JOGO DUROU " + (24 + fim - inicio) + " HORA(S)");
        }
    }
}

File: desafios/bee_1046.c
#include <stdio.h>
int main() {
    int inicio, fim;
    scanf ("%d %d", &inicio, &fim);
    if (inicio < fim) {
        printf("O JOGO DUROU %d HORA(S)\n", fim - inicio);
    }
    else if (inicio == fim) {
        printf("O JOGO DUROU 24 HORA(S)\n");
    }
    else {
        printf("O JOGO DUROU %d HORA(S)\n", 24 + fim - inicio);
    }
}